.github/workflows/ci.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     - name: Set up Python
       uses: actions/setup-python@v2
       with:
-        python-version: 3.10.7
+        python-version: 3.9
```

Comment:
This appear inconsistent with some changes in the other PR: https://github.com/marrlab/DomainLab/pull/743
Can both PRs coexist, or go you plan to discard one of the two?

Reply:
@agisga we need a PR that could let the CI pass, none of the two passes so we don't have to approve them?